### ISSUE TITLE
Enable support for eu-west-2

### ISSUE THREAD

**Issue by wonderphil** [spinnaker/spinnaker#1382]
*Title*

Enable support for eu-west-2 by updating Java AWS SDK
[clouddriver.txt](https://github.com/spinnaker/spinnaker/files/727083/clouddriver.txt)

*Cloud Provider*

AWS

*Environment*

I am running Spinnaker in AWS London Region, on ubuntu 14lts, with spinnaker-deck 2.974.0-3

*Feature Area*

CloudDriver.

*Description*

I should be able to have `SPINNAKER_AWS_DEFAULT_REGION=eu-west-2`, and run spinnaker. Currently if I use eu-west-2 as the region i get the following error

```
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [java.util.List]: Factory method 'synchronizeAmazonAccounts' threw exception; nested exception is java.lang.IllegalArgumentException: Cannot create enum from eu-west-2 value!
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:189)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:588)
	... 100 common frames omitted
Caused by: java.lang.IllegalArgumentException: Cannot create enum from eu-west-2 value!
	at com.amazonaws.regions.Regions.fromName(Regions.java:76)
	at com.netflix.spinnaker.clouddriver.aws.security.sdkclient.AwsSdkClientSupplier$AmazonClientKey.<init>(AwsSdkClientSupplier.java:163)
	at com.netflix.spinnaker.clouddriver.aws.security.sdkclient.AwsSdkClientSupplier.getClient(AwsSdkClientSupplier.java:65)
	at com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider.getAmazonEC2(AmazonClientProvider.java:249)
	at com.netflix.spinnaker.clouddriver.aws.security.DefaultAWSAccountInfoLookup.listRegions(DefaultAWSAccountInfoLookup.java:122)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader$1.get(CredentialsLoader.java:89)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader$1.get(CredentialsLoader.java:73)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader$Lazy.get(CredentialsLoader.java:263)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader.initRegions(CredentialsLoader.java:107)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader.load(CredentialsLoader.java:219)
	at com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader$load.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentialsInitializer.synchronizeAmazonAccounts(AmazonCredentialsInitializer.groovy:99)
	at com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentialsInitializer$$EnhancerBySpringCGLIB$$9a893587.CGLIB$synchronizeAmazonAccounts$8(<generated>)
	at com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentialsInitializer$$EnhancerBySpringCGLIB$$9a893587$$FastClassBySpringCGLIB$$e12d48a1.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:309)
	at com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentialsInitializer$$EnhancerBySpringCGLIB$$9a893587.synchronizeAmazonAccounts(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:162)
	... 101 common frames omitted
```

*Steps to Reproduce*

when spinnaker is running in eu-west-2, change `/etc/default/spinnaker` so that `SPINNAKER_AWS_DEFAULT_REGION=eu-west-2` then run `restart spinnaker` and check clouddriver logs

*Additional Details*

Additional information such as screenshots and exception logs.

**Comment by cfieber** [comment]
depends on Netflix/AWSObjectMapper#27

**Comment by chrono** [comment]
Any progress on this? Interested in support for `ca-central-1`.

**Comment by cfieber** [comment]
I expect this is fixed we updated to 1.11.90
